solutions/cpython-tests/readme.md

```diff
@@ -56,6 +56,8 @@ Here is a list of all the skipped individual unit tests
   * test_mkfifo_dir_fd
   * test_sched_rr_get_interval
   * test_sched_priority
+* test_poll
```

Comment:
Nit: align spacing

Reply:
Thats on purpose, its a separate category

Reply:
Ah, gotcha. My bad for viewing this in raw format!